packages/babel-plugin-transform-es2015-modules-commonjs/src/index.js

```diff
@@ -129,6 +129,10 @@ export default function () {
   return {
     inherits: require("babel-plugin-transform-strict-mode"),
 
+    manipulateOptions(opts, parserOpts) {
+      parserOpts.plugins.push("objectRestSpread");
+    },
```

Comment:
Would be better to inherits from `babel-plugin-syntax-object-rest-spread`. It seems multiple inheritance is not possible (at least didn't found an example).

To avoid that duplicated code, we could merge the config with [the syntax plugin](https://github.com/babel/babel/blob/8a82cc060ae0ab46bf52e05e592de770bd246f6f/packages/babel-plugin-syntax-object-rest-spread/src/index.js#L3-L5). What do you think?